rct229/rules/section6/section6rule3.py

```diff
@@ -0,0 +1,93 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import (
+    RuleDefinitionBase,
+    RuleDefinitionListIndexedBase,
+)
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.get_building_segment_lighting_status_type_dict import (
+    LightingStatusType,
+    get_building_segment_lighting_status_type_dict,
+)
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.std_comparisons import std_equal
+
+OFFICE_OPEN_PLAN = schema_enums[
+    "LightingSpaceType2019ASHRAE901TG37"
+].OFFICE_OPEN_PLAN.name
+
+
+class Section6Rule3(RuleDefinitionListIndexedBase):
+    """Rule 3 of ASHRAE 90.1-2019 Appendix G Section 6 (Lighting)"""
+
+    def __init__(self):
+        super(Section6Rule3, self).__init__(
+            rmrs_used=UserBaselineProposedVals(True, False, True),
+            each_rule=Section6Rule3.BuildingSegmentRule(),
+            index_rmr="baseline",
+            id="6-3",
+            description="Where a complete lighting system exists, the actual lighting power for each thermal block shall be used in the model. Where a lighting system has been designed and submitted with design documents, lighting power shall be determined in accordance with Sections 9.1.3 and 9.1.4. Where lighting neither exists nor is submitted with design documents, lighting shall comply with but not exceed the requirements of Section 9. Lighting power shall be determined in accordance with the Building Area Method (Section 9.5.1).",
```

Comment:
The description needs to be updated, here and in the RDS, since there are no longer "thermal blocks". I assume "building_segment" is the intent.